config/sync/views.view.taxonomy_term.yml

```diff
@@ -1,6 +1,6 @@
 uuid: d0072c03-96d6-4d54-964b-717cc6f08f4b
 langcode: en
-status: false
+status: true
```

Comment:
So this was disabled before as a whole. I've re-enabled it, disabled the unwanted displays and created a new embed display to use in the form class.